hook-consumer/src/config.rs

```diff
@@ -20,6 +20,9 @@ pub struct Config {
     #[envconfig(default = "5000")]
     pub request_timeout: EnvMsDuration,
 
+    #[envconfig(default = "1024")]
+    pub max_requests: usize,
```

Comment:
I'd call it `max_concurrent_requests` or `max_outstanding_requests` or something just to avoid any ambiguity.

Reply:
Oh, before I read your comment I went with `max_concurrent_jobs`. What do you think about that?

Reply:
I originally thought about capping only the HTTP requests to avoid hitting the file descriptor limit, hence the name.

Reply:
> Oh, before I read your comment I changed the name to `max_concurrent_jobs` in this commit: [31b1e7a](https://github.com/PostHog/rusty-hook/commit/31b1e7ae0e6f19fcf9cf40fd0e2a87e779667f19). What do you think about that?

Sounds good to me.